roles/manage_secrets/molecule/osp_secrets/converge.yml

```diff
@@ -0,0 +1,87 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
+    cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
+    cifmw_openshift_login_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
+    cifmw_manage_secrets_ospsecrets_list:
```

Comment:
Not sure how are we going to glue this with the rest? Thea idea is to generate this list in the new "edpm_prepare"?

Reply:
yeah, exactly. For now we have only two elements in that list (the `osp-secret` which contains password for every controlplane services and the `octavia-ca-passhprase` which contains, surprise surprise, the passphrase for its CA).
In our next iteration we'll have a secret for each service so we can handle their lifecycle in a more granular way.